Makefile

```diff
@@ -13,6 +13,13 @@ PKG_MODULES := vte-2.91
 PKG_CFLAGS  := $(shell pkg-config --cflags $(PKG_MODULES))
 PKG_LDLIBS  := $(shell pkg-config --libs   $(PKG_MODULES))
 
+prefix := $(PREFIX)
```

Comment:
I think it would be better to use `?=` instead of `:=` for variable assignments. That way it would be possible to override individual variables when invoking `make` — for example `make PREFIX=/ mandir=/usr/share/man` would work as expected. Could you please update the pull request with that? Thank you in advance :point_up: